### ISSUE TITLE
Installing prestashop locally on win10, last step reported error

### ISSUE THREAD

**Issue by stuffren** [PrestaShop/PrestaShop#35642]
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md).
- [X] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add attachments

Error details:
_Create file parameters
An error occurred during installation...
You can use the links on the left column to go back to the previous steps, or restart the installation process by [clicking here](http://localhost:8081/prestashop/install/index.php?restart=true).

1: HTTP 0 - error - undefined_

Screenshot:
![install_error](https://github.com/PrestaShop/PrestaShop/assets/136626456/ba193e60-2c8e-45a9-a632-cff45c167990)

Question 1: What usually causes this issue?
Question 2: What can I do to fix this issue?

My OS and server info is below:
------
OS: Windows 10 IoT Enterprise LTSC
Version number: 21H2
IDE: wampserver3.3.2-64bit
apache version: 2.4.58
php version: 8.3.0
mysql version: 8.2.0
phpmyadmin version: 5.2.1

### Expected behavior

Looking forward to successfully installing prestashop in win10!

### Steps to reproduce

1. During installation, it prompted me to enable SSL, I chose not to enable SSL.
2. Everything else is set up correctly, and the mysql database can be connected normally.
3. In the first step of installation, it prompts:
To avoid internationalization data inconsistencies upgrade the symfony/intl component.
I didn't solve this problem and just went to the next step!

### PrestaShop version(s) where the bug happened

I don't know if there is any relation!

### PHP version(s) where the bug happened

8.3.0

### If your bug is related to a module, specify its name and its version

I don't know if there is any relation!

**Comment by Codencode** [comment]
Duplicate of #35044

**Comment by Codencode** [comment]
Currently Prestashop does not support php 8.3.

Here you can see the system requirements: https://devdocs.prestashop-project.org/8/basics/installation/system-requirements/

**Comment by paulnoelcholot** [comment]
Hello @stuffren,

Unfortunately we cannot guarantee optimal use of PrestaShop on Windows.

https://devdocs.prestashop-project.org/8/basics/installation/system-requirements/

Thank you for understanding.

**Comment by stuffren** [comment]
> Hello @stuffren, 你好
> 
> Unfortunately we cannot guarantee optimal use of PrestaShop on Windows.很遗憾，我们无法保证在Windows上以最佳方式使用PrestaShop。
> 
> https://devdocs.prestashop-project.org/8/basics/installation/system-requirements/
> 
> Thank you for understanding.感谢您的理解。

Thank you for your reply, it doesn't matter!

**Comment by stuffren** [comment]
> Currently Prestashop does not support php 8.3.目前 Prestashop 不支持 php 8.3。
> 
> Here you can see the system requirements: https://devdocs.prestashop-project.org/8/basics/installation/system-requirements/在这里您可以看到系统要求： https://devdocs.prestashop-project.org/8/basics/installation/system-requirements/

Thank you for your reply。
Although PHP 8.3 is not supported, and I have tried other PHP versions, and I can't install it, but I think if I can't install it because it's not the right PHP version, it will look like Prestashop's limitation, and it will lose a lot of Prestashop's new users. As an out-of-the-box PHP application, it would be better to make the installation process very simple and easy!
Although I did end up successfully installing prestashop using docker.
But I sincerely hope that prestashop can do more and better with PHP versions and support for windows (10,11)!
